### PR TITLE
Ts 207: added an endpoint for the mail service so I can call it in the front end

### PR DESCRIPTION
**Pull request by GadoliniumPotassium**

### CODE REVIEW

**Comment by GadoliniumPotassium** [comment]
Can this be tested and merged so I can call this in the front end and finish my reset password PR

File: src/utils/email/MailController.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+class MailController {
+    private emailService:EmailService;
+    constructor(){
+        this.emailService = new EmailService();
+    }
+
+    async sendEmail(req:Request,res:Response){
+        try{
+            const mail = this.emailService.send(req.body);
```
Comment (kennguyen0303) [inline src/utils/email/MailController.ts:13]: We should verify the format of `req.body` to ensure all inputs suffice

File: src/utils/email/MailController.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+            return messages.successMessage('success','sentMail',mail)
```
Comment (kennguyen0303) [inline src/utils/email/MailController.ts:14]: Need to add `res.send(..)` to send the response to the client. If you don't need to send anything to the client, don't return a value.

```suggestion
            return res.send(messages.successMessage('success','sentMail',mail))
```

**Review by kennguyen0303 — CHANGES_REQUESTED** [review]
Looking good. Just need to

- [x] Verify the input in controller
- [x] register the route to the routeRegister

**Review by tungleu — APPROVED** [review]

**Review by tungleu — APPROVED** [review]

**Review by kennguyen0303 — APPROVED** [review]

**Pull request merged**